lib/Swagger/src/Discussion/Api/VotesApi.php

```diff
@@ -205,14 +194,15 @@ public function createUpVoteWithHttpInfo($site_id, $post_id)
                 $queryParams,
                 $httpBody,
                 $headerParams,
-                '\Swagger\Client\Discussion\Models\UpvoteHalResponse'
+                '\Swagger\Client\Discussion\Models\VotingResponse',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "\Swagger\Client\Discussion\Models\VotingResponse" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)